packages/allure-cypress/src/browser/utils.ts

```diff
@@ -0,0 +1,140 @@
+import { Status } from "allure-js-commons";
+import type { StatusDetails } from "allure-js-commons";
+import { extractMetadataFromString, getMessageAndTraceFromError, getStatusFromError } from "allure-js-commons/sdk";
+import type { CypressHook, CypressStepStopMessage, CypressSuite, CypressTest, StepDescriptor } from "../types.js";
+import { ALLURE_REPORT_SYSTEM_HOOK } from "./events/mocha.js";
+import { getAllureState, getProjectDir } from "./state.js";
+import { resolveStepStatus } from "./steps.js";
+
+export const getFileNameFromPath = (path: string) => path.substring(path.lastIndexOf(IS_WIN ? "\\" : "/") + 1);
```

Comment:
Those are browser-friendly (and limited) versions of getPosixPath + node:path functions, which I had to write because those from allure-js-commons are Node-only. I don't think other packages need them.

Also, I don't think we should use `slash`. It does the same replacing except a special handling for unprocessed Windows paths (`\\?\...`), which doesn't apply in our case (I don't think Cypress supports uses such paths). We would still need the branching on the platform (Unix file names can contain backslashes), so it doesn't save us much overall.